pywbem/mof_compiler.py

```diff
@@ -1627,7 +1633,10 @@ def p_instanceDeclaration(p):
             raise ce
 
     if alias:
+        if keybindings:
+            inst.path.keybindings = keybindings
```

Comment:
Here we are setting the path attribute of the instance we are building up. We need to figure out whether we want that. The logic is now: If an instance declaration in MOF has an alias definition, the compiled `CIMInstance` object will have its path set. otherwise, it will not have its path set. It seems to me that this is not particularly consistent.